code/controllers/subsystem/mapping.dm

```diff
@@ -329,10 +330,11 @@ SUBSYSTEM_DEF(mapping)
 		LAZYADD(SSzclear.free_levels, add_new_zlevel("Empty Area [space_levels_so_far]", ZTRAITS_SPACE, orbital_body_type = null))
 
 	// load mining
-	if(config.minetype == "lavaland")
-		LoadGroup(FailedZs, "Lavaland", "map_files/Mining", "Lavaland.dmm", default_traits = ZTRAITS_LAVALAND, orbital_body_type = /datum/orbital_object/z_linked/lavaland)
-	else if (!isnull(config.minetype))
-		INIT_ANNOUNCE("WARNING: An unknown minetype '[config.minetype]' was set! This is being ignored! Update the maploader code!")
+	if(SSmapping.map_adjustment ? !SSmapping.map_adjustment.no_lavaland : TRUE) // as long as it's allowed
```

Comment:
Altought it's unlikeley we will ever get another mining map, this would make lavaland the only option aviable for a mining map isn't it? i'd add this check after the `if(config.minetype == "lavaland")`

Reply:
`config.minetype = "lavaland"` is set by default. I guessed having another flag if we're going to use a mining planet or not would work.